yarn.lock

```diff
@@ -384,14 +384,6 @@
     "@typescript-eslint/typescript-estree" "4.29.3"
     debug "^4.3.1"
 
-"@typescript-eslint/scope-manager@4.29.2":
```

Comment:
Weird that there's diffs here given no change to package.json. Mind confirming and checking this in separately if these are due to a bad previous commit of yarn.lock?

Reply:
yeah, this is from a previous bad commit. without updating yarn.lock build is going to complain about unclean repo.